examples/functions-with-rest-parameter/functions_with_rest_parameter.description

```diff
@@ -1,3 +1,3 @@
-// Ballerina functions can have rest parameters. The rest parameter can take zero or more 
-// number of values. Inside the function, the rest parameter is equivalent to an array of 
+// Ballerina functions can have rest parameters. A rest parameter can take zero or more 
```

Comment:
Shall we reword it as below:
```
A Ballerina function can have at-most one rest parameter. This rest parameter can take zero or more values. The rest parameter inside a function is equivalent to an array of the same type.
```

please do correct any grammar, etc.. Just wanted to change the first line, since the original one itself is not technically correct..